Updated Figure documentation.

mattiasberlamont opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the styleguide CHANGELOG accordingly.

mattiasberlamont committed:
Updated Image documentation.

delrueba commented on components/31-molecules/figure/README.md:
```diff
@@ -1,20 +1,16 @@
 ## How it works
 
-The figure component wraps an image in a container and adds the benefit of providing an optional caption tag.
+The image component wraps an image in a container and adds the benefit of providing an optional caption tag.
```
```suggestion
The image component wraps an image in a figure and adds the benefit of providing an optional caption tag.
```

delrueba reviewed (changes requested):
Can you also add that the alt attribute is required and must be empty for purely decorative images.

There's a lot of content online about writing a good alt-text, but that might lead us too far, 
perhaps it's sufficient to reference the official WCAG Technique page: https://www.w3.org/TR/WCAG20-TECHS/H37.html

hstandaert reviewed (approved):
Might be useful to add some information about the fact that a ratio should always be provided, that we use `8:5` as a default and a placeholder image is loading or not found.

mattiasberlamont committed:
Update components/31-molecules/figure/README.md

Co-Authored-By: delrueba <31842807+delrueba@users.noreply.github.com>

mattiasberlamont committed:
Merge branch '3.x-dev' into feature/image-documentation

mattiasberlamont commented:
Should purely decorative images be wrapped in a figure?

mattiasberlamont committed:
Updated Figure documentation.

mattiasberlamont committed:
Updated Image Gallery documentation.

mattiasberlamont committed:
Updated Image Wall documentation.

mattiasberlamont commented:
@hstandaert @delrueba. After discussing this with @delrueba I think it would be better to keep the name "Figure" for this component. I updated the documentation and also added the requirements for images you mentioned. The references from the Image Gallery en Image Wall component documentation were also updated. Could you please review the changes?

delrueba reviewed (approved):
OK for me

mattiasberlamont commented:
@hstandaert OK for you?
